### PR TITLE
[WIP] API cleanup - ordering, filters

### PR DESCRIPTION
**Pull request by lechup**
* added ordering
* added filters
* other minor changes

WIP because I need to add much more tests (filtering + ordering, repetetive), please @khasinski @grillazz or @ajakubo1 take a look and say what do You think.

---
This change is [Reviewable](https://reviewable.io/reviews/enviromonitor/enviromonitorweb/54)

### CODE REVIEW

File: EnviroMonitorWeb/settings/base.py [hunk 1]
```diff
@@ -156,8 +156,12 @@
...
         'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
     ),
-    'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',),
+    'DEFAULT_FILTER_BACKENDS': (
+        'django_filters.rest_framework.DjangoFilterBackend',
+        'rest_framework.filters.OrderingFilter',
+    ),
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
     'PAGE_SIZE': 10,
+    'TEST_REQUEST_DEFAULT_FORMAT': 'json'
```
Comment (khasinski) [inline EnviroMonitorWeb/settings/base.py]: append `,`

File: api/tests/test_views.py [hunk 2]
```diff
@@ -115,6 +115,61 @@ def test_project_detail_delete(self):
...
+        api_response = self.client.get(
+            self.project_list_url,
+            data={
+                'owner': owner.pk
+            }
+        )
+        self.assertEqual(2, len(api_response.data['results']))
+
+    def test_list_view_filter_country(self):
+        country = 'Polska'
```
Comment (khasinski) [inline api/tests/test_views.py]: Shouldn't we use English names?

**Comment by lechup** [comment]
Review status: 0 of 5 files reviewed at latest revision, 2 unresolved discussions.

---

*[api/tests/test_views.py, line 146 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorweb/54#-KeI6tYYdsgoNLvfxZRI-r1-146:-KeSAkN5hEoPN9_DVMf_:b-nw1n9x) ([raw file](https://github.com/enviromonitor/enviromonitorweb/blob/5d56d7e3350f3576d080a931970856060e3b5636/api/tests/test_views.py#L146)):*
<details><summary><i>Previously, khasinski (Krzysztof Hasiński) wrote…</i></summary><blockquote>

Shouldn't we use English names?
</blockquote></details>

Yeah, You are right...

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/enviromonitor/enviromonitorweb/54)*

**Comment by grillazz** [comment]
Reviewed 3 of 5 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/enviromonitor/enviromonitorweb/54)*

**Pull request merged**